Sources/tart/Commands/Create.swift

```diff
@@ -19,6 +19,9 @@ struct Create: AsyncParsableCommand {
   @Option(help: ArgumentHelp("Disk size in GB"))
   var diskSize: UInt16 = 50
 
+  @Option(help: ArgumentHelp("Disk image format: raw (default) or asif", discussion: "ASIF format provides better performance but requires macOS 26 Tahoe or later"))
```

Comment:
`ArgumentParser` already shows the possible values and the default value:

```suggestion
  @Option(help: ArgumentHelp("Disk image format", discussion: "ASIF format provides better performance but requires macOS 26 Tahoe or later"))
```